docs/admin/high-availability/index.md

```diff
@@ -194,7 +194,7 @@ touch /var/log/kube-scheduler.log
 touch /var/log/kube-controller-manager.log
 ```
 
-Next, set up the descriptions of the scheduler and controller manager pods on each node.
+Next, set up the descriptions of the scheduler and controller manager pods on each node,

```

Comment:
You can remove the punctuation here, so it should be: "Next, set up the descriptions of the scheduler and controller manager pods on each node by copying [kube-scheduler.yaml](/docs/admin/high-availability/kube-scheduler.yaml) and ..."